lib/Cargo.toml

```diff
@@ -12,6 +12,15 @@ env_logger = "0.8.4"
 directories = "5.0.1" # For Environment to find platform-specific config location
 smallvec = "1.10.0"
 
+[dependencies.uuid]
+version = "1.5.0"
+features = [
+    "v4",                # Lets you generate random UUIDs
+    "fast-rng",          # Use a faster (but still sufficiently random) RNG
+    "macro-diagnostics", # Enable better diagnostics for compile-time UUIDs
+]
```

Comment:
A very minor point, but style-wise personally prefer to use the more compact notation unless there was a particular reason you want a comment explaining your choice of features.

e.g. `uuid = {version="1.5.0", features=["v4", "fast-rng", "macro-diagnostics"]}`

Also, uuid v 1.6.x has been released.  https://crates.io/crates/uuid